.github/workflows/test.yaml

```diff
@@ -0,0 +1,9 @@
+name: test
+on: push
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run tests
+        run: ./test.sh --is-ci
```

Comment:
Seems like this workflow is not running. I wonder if we need to _enable_ anything on this repo settings page.

Reply:
I checked the repo settings and I'm not seeing anything that would block a workflow from being recognized.  I tried a couple updates to the test.yaml file and none of them worked either. Maybe the workflow has to be present on the branch with the PR on it, rather than the upstream branch?

Reply:
I think it's a problem with forks. Check this out, https://github.community/t/actions-not-working-correctly-for-forks/16649/5.

Reply:
oh that's so frustrating!

Reply:
could we get around it with CircleCI instead?